monai/data/image_reader.py

```diff
@@ -107,47 +110,48 @@ def verify_suffix(self, filename: Union[Sequence[str], str]) -> bool:
         """
         return True
 
-    def read(self, data: Union[Sequence[str], str, Any]):
+    def read(self, data: Union[Sequence[str], str], **kwargs):
         """
-        Read image data from specified file or files, or set a `itk.Image` object.
+        Read image data from specified file or files.
         Note that the returned object is ITK image object or list of ITK image objects.
-        `self._img` is always a list, even only has 1 image.
 
         Args:
-            data: file name or a list of file names to read, or a `itk.Image` object for the usage that
-                the image data is already in memory and no need to read from file again.
+            data: file name or a list of file names to read,
+            kwargs: additional args for `itk.imread` API, if not set, using `self.kwargs`.
+                More details about available args:
+                https://github.com/InsightSoftwareConsortium/ITK/blob/master/Wrapping/Generators/Python/itkExtras.py
 
         """
-        self._img = list()
-        if isinstance(data, Image):
-            self._img.append(data)
-            return data
+        img_: List[Image] = list()
 
         filenames: Sequence[str] = ensure_tuple(data)
+        if len(kwargs) == 0:
+            kwargs = self.kwargs
```

Comment:
`if len(kwargs) ==0: kwargs = self.kwargs`
could be: `self.kwargs.update(kwargs)`, so that `kwargs` could overwrite `self.kwargs` and it becomes more flexible

Reply:
Hi @wyli ,

I changed to `args = self.kwargs.copy(), args.update(kwargs)` because I think the `kwargs` in `read()` is 1 time call, should not modify the member property of the reader object, and also for thread safe, let's don't change properties.

Thanks.